docs/resources/service_object.md

```diff
@@ -1,19 +1,17 @@
 # valtix_service_object
-Service objects represent the backend application and how it's serviced to the external users.
-Valtix gateway acts as a proxy to this service and all the end users' access to the application
-goes via the gateway. The service defines a listen port where the gateway listens and specifies
-the target ip/fqdn and port number to which the end users' requests are forwarded. If the listen
-port must be TLS encrypted, a decryption profile with a certificate must be provided.
-The transport to the backend can be a HTTP, HTTPS, TCP or TLS session. Please look into the
-documentation for the details of each of these types.
-If multiple services (backends) need to listen on the same port (for e.g port 443), then an SNI
+Service objects represent the backend application and how it's serviced to the external users. The Valtix Gateway acts as a proxy to this service and all the end users' access to the application passes through the Gateway. The service defines a listener port and a backend target IP/FQDN and port number to which the end users' requests are forwarded. If the traffic is TLS encrypted and there is a desire to decrypt the traffic to inspect using advanced security, a Decryption Profile with a Certificate must be provided.  The transport to the backend can be a HTTP, HTTPS, TCP or TLS session. Please look into the documentation for the details of each of backend transport types.
```

Comment:
May need correction. Service object is not always a backend, its also a bunch of ports. The above description would be suitable for the reverseproxy object